cvm/quantization/from_tensorflow.py

```diff
@@ -592,10 +633,15 @@ def convert_tfnode(tfnode, graph, params, infer_shapes, logger=logging):
             elif k not in ('dtype', '_output_shapes', '_class'):
                 raise NotImplementedError \
                     ("Other attributes for a Const(param) Node {} ? .".format(k))
-    elif op_name in ['Placeholder', 'PlaceholderWithDefault']:
+    elif op_name in ['Placeholder', 'PlaceholderWithDefault', 'FIFOQueueV2']:
         input_shape = \
             tensor_util.tensor_shape_proto_to_list(attr['shape'].shape)
-        dtype = tensor_util.tensor_type_to_numpy(attr['dtype'].type)
+        if not input_shape:
+            input_shape = default_input_shape
+        try:
+            dtype = tensor_util.tensor_type_to_numpy(attr['dtype'].type)
+        except TypeError:
+            dtype = tensor_util.tensor_type_to_numpy(default_tf_dtype)
```

Comment:
Why this exception of TypeError? can you explain a lot more via comments?

Reply:
Ususally in tensorflow models, attr['dtype'].type will get a value of 1, which represents float32 in tensorflow. 
However in resnet_v2_101, attr['dtype'].type will get a value of 0, and TypeError occurs. set the value to default_tf_dtype would work fine.

Reply:
Then what the value of 0 means, do you know?

Reply:
0 means `DT_INVALID`

Usage:
indicate a DataType field has not been set.

Declaration:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/types.proto

Which models have:
ssd_mobilenet_v2_coco, resnet_v2_101_lite
the first two layers of both models are repectively: FIFOQueueV2, QueueDequeueManyV2
I don't know how to realize this two layer in mxnet.

My solution:
fuse first two layer into an input layer, and directly set the data type as DT_FLOAT which is 1

Reply:
Correctness: only resnet_v2_101_lite, ssd_mobilenet_v2_coco is not.

Reply:
En, It seems the operators: FIFOQueueV2, QueueDequeueManyV2 that brings into DT_INVALID problem. I'll try to solve it when I'm free and this PR is stuck here temporarily.

Reply:
@declmal , Add TODO in the source code at here, I'll merge the PR 57 first before [PR 58](https://github.com/CortexFoundation/tvm-cvm/pull/58)